src/codec/h264/parser.rs

```diff
@@ -647,7 +647,7 @@ pub struct Sps {
 
     /// Plus 1 specifies the width of each decoded picture in units of
     /// macroblocks.
-    pub pic_width_in_mbs_minus1: u32,
+    pub pic_width_in_mbs_minus1: u8,
```

Comment:
> The H.264 specification mentions that the maximum width of a stream is
4096 pixels. This means that pic_width_in_mbs_minus1 cannot exceed 255,
so change its type to u8 and reject input values that won't fit.

Could you please provide a source for this? I am afraid it's actually 8192x4320 for level 6.2 according to Table A-6.

Reply:
You're right, my information was outdated. Let's use u16 for width and height, it still fixes the overflow issue I wanted to avoid anyway.